src/main/routes/scenario.js

```diff
@@ -0,0 +1,73 @@
+import express from 'express';
+const router = express.Router();
+import { Scenario } from '../modules/Scenario';
+import { Execution } from '../modules/Execution';
+import { Output } from '../modules/Output';
+import { ScenarioVerifier } from '../api-utilities/ScenarioVerifier';
+import { TemplateVerifier } from '../api-utilities/TemplateVerifier';
+import { ActorsOfActions } from '../api-utilities/SinglePropertyVerifier';
+import admin from 'firebase-admin';
+import serviceAccount from '../../../../claves/metal-slug-maker-firebase-adminsdk-8dsbv-e517ebfe1a.json';
```

Comment:
Since Samuel changed the README file with a different way to include the ServiceAccountkey please change the name of the folder "claves" to "keys"